datahub-web-react/src/app/entity/user/UserProfile.tsx

```diff
@@ -1,110 +1,411 @@
+                },
             },
         ].filter((tab) => ENABLED_TAB_TYPES.includes(tab.name));
     };
+    const defaultTabPath = getTabs() && getTabs()?.length > 0 ? getTabs()[0].path : '';
+    const onTabChange = () => null;
 
-    const getHeader = (user: CorpUser) => {
-        const { editableInfo, info } = user;
-        const displayName = entityRegistry.getDisplayName(EntityType.CorpUser, user);
-        return (
-            <UserHeader
-                profileSrc={editableInfo?.pictureLink}
-                name={displayName}
-                title={info?.title}
-                email={info?.email}
-                skills={editableInfo?.skills}
-                teams={editableInfo?.teams}
-            />
-        );
+    // EditProfile modal Constants
+    const getEditModalData = () => {
+        return {
+            urn: data?.corpUser?.urn || undefined,
+            name: data?.corpUser?.info?.fullName || undefined,
+            title: data?.corpUser?.info?.title || undefined,
+            team: data?.corpUser?.editableProperties?.teams?.join(',') || undefined,
+            email: data?.corpUser?.info?.email || undefined,
+            image: profileSrc,
+            slack: data?.corpUser?.editableProperties?.slack || undefined,
+            phone: data?.corpUser?.editableProperties?.phone || undefined,
+        };
     };
 
+    // About Text Change
+    const aboutMeTextChangeFunction = (inputString) => {
+        setEditableAboutMeText(inputString);
+        updateCorpUserPropertiesMutation({
+            variables: {
+                urn: data?.corpUser?.urn || '',
+                input: {
+                    aboutMe: inputString,
+                },
+            },
+        })
+            .catch((e) => {
+                message.destroy();
+                message.error({ content: `Failed to Save changes!: \n ${e.message || ''}`, duration: 3 });
+            })
+            .finally(() => {
+                message.success({
+                    content: `Changes saved.`,
+                    duration: 3,
+                });
+                refetch();
+            });
+    };
     return (
         <>
             {contentLoading && <Message type="loading" content="Loading..." style={messageStyle} />}
-            {data && data.corpUser && (
-                <LegacyEntityProfile
-                    title=""
-                    tags={null}
-                    header={getHeader(data.corpUser as CorpUser)}
-                    tabs={getTabs()}
+            <UserProfileWrapper>
+                <Row>
+                    <Col xl={5} lg={5} md={5} sm={24} xs={24}>
+                        <UserSidebar>
+                            <UserSidebarSubSection>
+                                <CustomAvatar
+                                    size={160}
+                                    photoUrl={profileSrc}
+                                    name={data?.corpUser?.info?.fullName || undefined}
```

Comment:
Get the name from the displayname which should be this:

`data?.corpUser.editableInfo?.displayName || data?.corpUser?.info?.displayName || data?.corpUser?.info?.fullName || data?.corpUser.urn`

Reply:
sure, required changes are added